erpnext/accounts/report/accounts_receivable/accounts_receivable.html

```
@@ -1,291 +1,225 @@
-						{% if(!filters.show_future_payments) { %}
-							<td style="text-align: right">
-								{%= format_currency(data[i]["paid"], data[i]["currency"]) %}</td>
-							<td style="text-align: right">
-								{%= format_currency(data[i]["credit_note"], data[i]["currency"]) %}</td>
-						{% } %}
-						<td style="text-align: right">
-							{%= format_currency(data[i]["outstanding"], data[i]["currency"]) %}</td>
+	{% if(filters.show_future_payments) { %}
+		{%
+			var balance_row = data.slice(-1).pop();
+			var start = report.columns.findIndex(e => e.fieldname == 'age');
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)